README.md

```diff
@@ -0,0 +1,56 @@
+# vpphelper
+
+vpphelper provides helper functions (using govpp) that helps to connect to VPP and configure VPP using a configuration (.conf) file.
+
+## StartAndDialContext
+`StartAndDialContext` function starts VPP using a VPP configuration file and returns a `Connection` to VPP. 
+
+```go
+conn, vppErrCh := vpphelper.StartAndDialContext(connectCtx)
+```
+```go
+func StartAndDialContext(ctx context.Context, opts ...Option) (conn Connection, errCh <-chan error) {
+	o := &option{
+		rootDir:   DefaultRootDir,
+		vppConfig: vppConfContents,
+	}
+	for _, opt := range opts {
+		opt(o)
+	}
+
+	if err := writeDefaultConfigFiles(ctx, o); err != nil {
+		errCh := make(chan error, 1)
+		errCh <- err
+		close(errCh)
+		return nil, errCh
+	}
+	logWriter := log.Entry(ctx).WithField("cmd", "vpp").Writer()
+	vppErrCh := exechelper.Start("vpp -c "+filepath.Join(o.rootDir, vppConfFilename),
+		exechelper.WithContext(ctx),
+		exechelper.WithStdout(logWriter),
+		exechelper.WithStderr(logWriter),
+	)
+	select {
+	case err := <-vppErrCh:
+		errCh := make(chan error, 1)
+		errCh <- err
+		close(errCh)
+		return nil, errCh
+	default:
+	}
+
+	return DialContext(ctx, filepath.Join(o.rootDir, "/var/run/vpp/api.sock")), vppErrCh
+}
+```
+
+* If no options are passed to `StartAndDialContext` function, then the `rootDir` is set to `DefaultRootDir` constant (initialized in `options.go`) and `vppConfig` is set to `vppConfContents` constant (initialized in `vpp.conf.go`).
```

Comment:
It might be useful documenting each Option here.

Reply:
Ok I will add it.